lib/NativeCall.pm

```diff
@@ -22,11 +22,11 @@ sub _attr_parse {
     (\w+)
     (?:
       \(
-      (.*?)
+      (.*)
```

Comment:
Doesn't this change make it greedy-match all the way to the last bracket? Everything else looks great, just want a bit of help understanding this.